Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -1654,27 +1654,15 @@ bool CStaticFunctionDefinitions::SetElementHealth(CElement* pElement, float fHea
         case CElement::PLAYER:
         {
             CPed* pPed = static_cast<CPed*>(pElement);
-            if (pPed->IsSpawned())
-            {
-                // Limit their max health to what the stat says
-                float fMaxHealth = pPed->GetMaxHealth();
-                if (fHealth > fMaxHealth)
-                    fHealth = fMaxHealth;
+            if (!pPed->IsSpawned())
+                return false;
 
-                // Do not set the health below zero
-                if (fHealth < 0.0f)
-                    fHealth = 0.0f;
+            fHealth = Clamp(0.0f, fHealth, pPed->GetMaxHealth());
+            pPed->SetHealth(fHealth);
 
-                // This makes sure the health is set to what will get reported
-                unsigned char ucHealth = static_cast<unsigned char>(fHealth * 1.25f);
-                fHealth = static_cast<float>(ucHealth) / 1.25f;
```

Comment:
Why did you remove `1.25f` coefficient?

Reply:
Because it was completely pointless and caused this issue to happen